block/manager.go

```diff
@@ -261,9 +269,38 @@ func NewManager(
 		metrics:       seqMetrics,
 		isProposer:    isProposer,
 	}
+	agg.init(context.Background())
```

Comment:
shall we maybe pass WithCancel? Or no need right now?

Reply:
We're using background for most store tasks, maybe we can visit this later.